tools/pipelines/build-protocol-definitions.yml

```diff
@@ -41,6 +41,7 @@ trigger:
     - tools/pipelines/templates/build-npm-package.yml
     - tools/pipelines/templates/include-set-package-version.yml
     - tools/pipelines/templates/include-vars.yml
+    - tools/pipelines/templates/include-install-pnpm.yml
```

Comment:
(Not really for this PR, but a more general question) The list of templates we have to specify in our pipelines has gotten pretty large. Should we just include `tools/pipelines/templates` and eat the cost of potentially unnecessary runs to reduce the overhead / reduce the risk of missing things?